src/main/java/org/snomed/snowstorm/core/data/services/ConceptChangeHelper.java

```diff
@@ -255,128 +255,153 @@ private NativeSearchQueryBuilder newSearchQuery(BoolQueryBuilder branchUpdatesCr
      * the other Branch.
      */
     public Set<Long> getContradictoryChanges(Branch source, Branch target) {
-        Set<Long> conceptsDeletedOnBranch = getConceptsDeletedOnBranch(target);
+        Set<String> conceptsDeletedOnBranch = getConceptsDeletedOnBranch(target);
         if (conceptsDeletedOnBranch.isEmpty()) {
             return Collections.emptySet();
         }
 
-        return getConceptsWithModifiedDescriptionsOnBranch(source, conceptsDeletedOnBranch);
+        Set<String> conceptsWithModifiedDescriptionsOnBranch = getConceptsWithModifiedDescriptionsOnBranch(source, conceptsDeletedOnBranch);
```

Comment:
Maybe we should use source for the first method call e.g getConceptsDeletedOnBranch(source) and getConceptsWithModifiedDescriptionsOnBranch(target)

Reply:
A couple of tests failed with making this change. Flipping the branches might require a re-work of the implementation. Does this need doing or are we happy with querying `target` first?

Reply:
hmm I wasn't expecting tests to fail and will take a look together tomorrow.